setup.py

```diff
@@ -1,26 +1,29 @@
-import os
-
 try:
-  from setuptools import setup
-except:
-  from distutils.core import setup
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
 
 
-setup(name = "clrsvsim",
-      version = "0.0.2",
-      description = "Color Genomics Structural Variant Simulator",
-      author = "Color Genomics",
-      author_email = "dev@color.com",
-      url = "https://github.com/ColorGenomics/clrsvsim",
-      packages = ["clrsvsim"],
-      install_requires=[
-        'cigar==0.1.3',
-        'mock==2.0.0',
-        'nose==1.3.7',
-        'numpy==1.10.1',
-        'preconditions==0.1',
-        'pyfasta==0.5.2',
-        'pysam==0.10.0',
-      ],
-      license = "Apache-2.0",
+setup(
+    name="clrsvsim",
+    version="0.1.0",
+    description="Color Genomics Structural Variant Simulator",
+    author="Color",
+    author_email="dev@color.com",
+    url="https://github.com/color/clrsvsim",
+    packages=["clrsvsim"],
+    install_requires=[
+        "cigar>=0.1.3",
+        "numpy>=1.10.1",
+        "preconditions>=0.1",
+        "pyfasta>=0.5.2",
+        "pysam>=0.10.0",
```

Comment:
Consider following semver dependency management best practice by pinning the major version only, i.e. `package >= 1.2.3, < 2`

Reply:
Agreed, in the longer run that's probably exactly what we want to do. For now, this package's use of pysam is so minimal that I'm not sure an eventual 1.0 release would be likely to break functionality.

May revisit at a later time!

Reply:
Oh I'm talking in general, not specific to pysam. But not a blocking issue.

Reply:
Yup, totally agreed: this package should be pinning to the same major version for _each_ of these dependencies.

Should have been more clear there. All I mean to say is that for pysam (& others) no new major releases are on the horizon yet, nor do we really cleanly catalog/test for what we expect from deps yet.

In the future we should be better about our pinning.